src/data_analysis.py

```diff
@@ -0,0 +1,47 @@
+import pandas as pd
+
+
+df = pd.read_csv('data/task3_dateset.csv', dtype=str)
+df['Date'] = pd.to_datetime(df['Date'], format='%m/%d/%y')
+df['Ads_Run'] = df['Ads_Run'].fillna(0)
+df['Ads_Run'] = df['Ads_Run'].astype(int)
+df = df.sort_values(by=['Site', 'Date'])
```

Comment:
lines 5-8 should really be in the fix() function at: https://github.com/markfyoung0711/coop-market-2/pull/4/files#diff-3de666883916d7116c094e3ad4a76b5ddfa2dc5f424918afc59e9b7fd6dca85cR17

This takes the onus off of the client of fix() to correct the date and ensures Ads_Run type is an int